### ISSUE TITLE
Chapter 5 - error in formula

### ISSUE THREAD

**Issue by yonigottesman** [nlp-with-transformers/notebooks#92]
## Information

The problem arises in chapter:

* [ ] Introduction
* [ ] Text Classification
* [ ] Transformer Anatomy
* [ ] Multilingual Named Entity Recognition
* [x] Text Generation
* [ ] Summarization
* [ ] Question Answering
* [ ] Making Transformers Efficient in Production
* [ ] Dealing with Few to No Labels
* [ ] Training Transformers from Scratch
* [ ] Future Directions

## Describe the bug
![image](https://user-images.githubusercontent.com/4004127/224643814-def2f366-a2e4-4555-b264-149be387b476.png)
I think the softmax is calculated on z_t, and i is taken on the result of the softmax. not like in the book where the softmax is calculated on a single logit t,i which doesnt make sense, softmax needs all the "i"s 

## To Reproduce

1.
2.
3.
